static_src/util/cf_api.js

```diff
@@ -262,7 +262,7 @@ export default {
   postAppRestart(appGuid) {
     return http.post(`${APIV}/apps/${appGuid}/restage`).then(() => {
       appActions.restarted(appGuid);
-    });
+    }).catch((err) => handleError(err, appActions.error.bind(this, appGuid)));
```

Comment:
The `handlerError` function takes care of some data handling in all error cases: https://github.com/18F/cg-dashboard/pull/748/files/3c8e204600f8c1ddd92c2b848f9f1170f59899a4#diff-d45ddca2d392b7558d8957b40dd036ccL19.